packages/web-demo/src/app/bootstrap/components/form-select/form-select-demo/form-select-demo.component.html

```diff
@@ -1,10 +1,17 @@
 <div class="form-floating">
-  <select aria-label="Floating label select example" class="form-select form-select-lg" id="floatingSelect">
+  <select aria-label="Floating label select example" class="form-select" id="floatingSelect">
     <option value="1">One</option>
     <option value="2">Two</option>
     <option value="3">Three</option>
   </select>
-  <label for="floatingSelect">.form-select-lg</label>
+  <label for="floatingSelect">Floatinglabel form-select</label>
+</div>
+
+<div class="form-floating">
+  <select aria-label="Floating label select example" class="form-select" id="floatingSelectEmpty">
+    <!-- no options -->
+  </select>
```

Comment:
I think the example with no selected option for the custom select is enough to show the behaviour. I'd rather not put bad examples like selects without options in the docs. They look broken. If you agree, can you remove the two examples without options?